### PR TITLE
added models for skills collection

### PR DESCRIPTION
**Pull request by Mir-SA**
Adds Skill model for its API
Adds Skills property on User Object

[Reference](https://github.com/Real-Dev-Squad/website-api-contracts/issues/70) for its contract

### CODE REVIEW

File: skills/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Skills
+Firestore Skills collection data model.
+
+```
+{
+    name: String,
+    awarded_on: String,
```
Comment (ankushdharkar) [inline skills/README.md:7]: Can this just be 'on', 'by', and 'for'?

File: users/README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
   'linkedin_id': string,
   'twitter_id': string,
   'instagram_id': string,
+  'skills': [],
```
Comment (ankushdharkar) [inline users/README.md:19]: What is the size limit of a document in firestore? What happens if we add too many skills? How will we search using skills in future if we add everything to an array?

Say, I want to find out "Show all users having skill 'Node-Level-Black', what will the query look like?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: skills/README.md (hunk 1 above)
Comment (Mir-SA) [reply skills/README.md:7]: sure.

File: users/README.md (hunk 2 above)
Comment (Mir-SA) [reply users/README.md:19]: 1. I didn't really keep the limit in mind...
2. we can fetch users and check which user/s have that skill in their skills prop

**Comment by Mir-SA** [comment]
@ankushdharkar I've made suggested changes.
Please check them and lmk your thoughts.

**Review by ankushdharkar — APPROVED** [review]

**Comment by prakashchoudhary07** [comment]
Reverting this PR, as this feature is being built as a stand-alone product, in Skill-tree

https://github.com/Real-Dev-Squad/skill-tree-frontend/
https://github.com/Real-Dev-Squad/skill-tree-backend/

**Pull request merged**